exporters/logging-otlp/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'OpenTelemetry Protocol JSON Logging Exporters'
+ext.moduleName = "io.opentelemetry.exporter.logging.otlp"
```

Comment:
my only possible quibble is that maybe this should be `"io.opentelemetry.exporter.logging-otlp"` but I'm not quite sure which is going to be more discoverable by end-users. Maybe the description could be more .. descriptive about exactly what's going on here.  Maybe `description = 'OpenTelemetry Logging exporters with OTLP JSON formatting'` ?